PhysicsTools/HepMCCandAlgos/python/genPUProtons_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+genPUProtons = cms.EDProducer("GenPUProtonProducer",
+	mix = cms.string("mix"),
+	bunchCrossingList = cms.vint32(0),
+	minPz = cms.double( 4800. )
```

Comment:
should these also cover
- pA runs
- lower energy pp runs
    - recall that the default config is still for run1 which appears to be not covered by this PR